ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Net;
+using System.Text.Json.Nodes;
+using System.Text;
+using System.Linq;
+using System;
+using System.Runtime.InteropServices.ComTypes;
+using WasmBenchmarkResults;
+
+class Test
+{
+    private static string getFlavor(string line)
+    {
+        var words = line.Split("/");
+        StringBuilder stringBuilder = new();
+        for (var i = 2; i < words.Length - 1; i++)
+        {
+            stringBuilder.Append(words[i] + ".");
+        }
+
+        return stringBuilder.ToString().Remove(stringBuilder.Length - 1);
+    }
+
+    static async Task Main(string[] args)
+    {
+        var client = new HttpClient();

```

Comment:
nitpick: you might use the `new()` here as well to be consistent with the below lines.

Reply:
Will be visible in the next commit